src/processors/default/addTimestampToERC721NFTs.ts

```diff
@@ -7,13 +8,14 @@ import { Record } from '../../types/record';
 
 const processorFunction = (table: Table) => async (items: Record, clients: Clients) => {
   const recordsByBlockNumber = _.groupBy(items, 'createdAtEthereumBlockNumber');
-  const blockNumbers = Object.keys(recordsByBlockNumber);
+  const blockNumbers = Object.keys(recordsByBlockNumber).map(blockNumber => Number.parseInt(blockNumber));
   console.log(`Processing until block ${blockNumbers[blockNumbers.length - 1]}`)
-  const timestampsResponse = await clients.blocks.fetchBlockTimestamps(blockNumbers);
```

Comment:
Looks like we're not using `fetchBlockTimestamps` anywhere else, so we can likely remove the BlocksClient from the list of `Clients`, and then also nuke `blocks.ts` and the imports associated with it.

Also the `BLOCKS_SUBGRAPH_ENDPOINT` goes away! One less external dependency!